### PR TITLE
Feature prior

### LINKED ISSUE #17: Specify format for priors in parameter table

### PR DESCRIPTION
**Pull request by paulstapor**
Added some functionality to sample starting points according to priors, added files for a test (Fixes #17 )

### CODE REVIEW

File: petab/core.py [hunk 1]
```diff
@@ -741,6 +749,128 @@ def append_overrides(overrides):
...
+
+    elif p_type == 'parameterScaleUniform':
+        sp = (p_params[1] - p_params[0]) * np.random.random((n_starts,
+                                                             )) + p_params[0]
+
+    elif p_type == 'normal':
+        sp = scale(np.random.normal(loc=p_params[0], scale=p_params[1],
+                                    size=(n_starts,)))
+
+    elif p_type == 'logNormal':
```
Comment (jvanhoefer) [inline petab/core.py]: We need to decide on a consistent parametrization of the distributions. 
I understood from our discussion (might be my misunderstanding...), that we agreed on another parametrization, but I prefer your parametrization here. What do you think @LeonardSchmiester? :)

Comment (paulstapor) [reply petab/core.py]: I thought that was the way we agreed on it: Introducing log-normal as an exponentiated normal distribution on the linear parameter space? What should be different?

Comment (jvanhoefer) [reply petab/core.py]: I thought that we parametrized it as it is parametrized in scipy. But as I said, I prefer your version and just wanted to make sure, that Leonard implements the same parametrization. I will adapt my code accordingly.

**Comment by dweindl** [comment]
@paulstapor : Can you please include a section in the documentation on the (meaning of the) supported values for the respective columns?

**Comment by paulstapor** [comment]
> @paulstapor : Can you please include a section in the documentation on the (meaning of the) supported values for the respective columns?

Indeed, that's still missing. Will do that before merging!

File: petab/core.py (hunk 1 above)
Comment (paulstapor) [reply petab/core.py]: Ah...
Yes, scipy does it differently...
Yes, but we do everything via numpy here, so I would also prefer not to get involved with yet another package which has redundant functionality, but uses a different definition...

**Comment by paulstapor** [comment]
Okay, I updated the documentation, also about the additional columns we're actually still discussing about (initializationPriorType, initializationPriorParameters, objectivePriorType, and objectivePriorParameters). This is yet to come, but fairly simple to implement. I think/hope, it was safe to include this in the "Extensions" part of the documentation...

**Comment by paulstapor** [comment]
Can we agree on that this fixes now #17 ?

**Comment by dweindl** [comment]
> Can we agree on that this fixes now #17 ?

Okay for me.

**Comment by paulstapor** [comment]
May/Can/Shall I merge?
Or should I first fix #131 ?

**Comment by paulstapor** [comment]
> > May/Can/Shall I merge?
> > Or should I first address #131 ?
> 
> Thats a PEtab issue. Merging in pesto should be done independently of that.

Both things (this PR and the Issue) are PEtab things...
I don't see, where pypesto comes in here...

**Pull request merged**